pkg/csi/service/common/util.go

```diff
@@ -436,3 +437,37 @@ func GetCSINamespace() string {
 	}
 	return CSINamespace
 }
+
+func GetValidatedCNSVolumeInfoPatch(ctx context.Context, cnsSnapshotInfo *cnsvolume.CnsSnapshotInfo, cnsVolumeInfo *cnsvolumeinfov1alpha1.CNSVolumeInfo) map[string]interface{} {
+	log := logger.GetLogger(ctx)
+	var patch map[string]interface{}
+	if cnsSnapshotInfo.AggregatedSnapshotCapacityInMb == -1 {
+		log.Infof("Couldn't retrieve aggregated snapshot capacity for volume %d and snapshot %d",
+			cnsVolumeInfo.Spec.VolumeID, cnsSnapshotInfo.SnapshotID)
+		patchAnnotation := MergeMaps(cnsVolumeInfo.Annotations,
+			map[string]string{"csi.vsphere.missing-snapshot-aggregated-capacity": "true"})
```

Comment:
Please add "csi.vsphere.missing-snapshot-aggregated-capacity" in a constant variable, as it is getting reused at multiple places